techniques/html/H43.html

```diff
@@ -100,66 +84,57 @@ <h3>Example 1: A table with multiple rows of headers</h3><pre xml:space="preserv
   &lt;/table&gt;</pre></section>
          </section>
          <section id="resources">
-            <h2>Resources</h2>
-            <p>Resources are for information purposes only, no endorsement implied.</p>
+            <h2>参考リソース</h2>
+            <p>この参考リソースは、あくまでも情報提供のみが目的であり、推薦などを意味するものではない。</p>
             <ul>
                
                <li>HTML 4.01 <a href="https://www.w3.org/TR/html4/struct/tables.html#h-11.4.1">header information with data cells</a>
-                  								       
                </li>
                
             </ul>
+            <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注:</div><div><p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。上記は、HTML 5.2§4.9.10. The th element を代わりに参照できる。</p></div></div>
```

Comment:
HTML Standardへのリンクを追加します。
```suggestion
            <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注:</div><div><p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。上記は、<a href="https://html.spec.whatwg.org/multipage/tables.html#the-th-element">HTML §4.9.10. The th element</a> を代わりに参照できる。</p></div></div>
```